configure.ac

```diff
@@ -201,6 +186,7 @@ AC_ARG_WITH(cmodel,
 	[],
 	[]
 	)
+
 AS_IF([test "x$with_cmodel" != x],
```

Comment:
This is still buggy for `--without-cmodel`. Would be great if you could fix this as well.